package/gluon-status-page-mesh-batman-adv/Makefile

```diff
@@ -0,0 +1,23 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=gluon-status-page-mesh-batman-adv
+PKG_VERSION:=1
+
+include ../gluon.mk
+
+define Package/gluon-status-page-mesh-batman-adv
+  TITLE:=Batman-data provider for gluon-status-page
+  SECTION:=gluon
+  CATEGORY:=Gluon
+  PROVIDES:=gluon-status-page-data-provider
+  DEPENDS:=+gluon-status-page +gluon-mesh-batman-adv +libnl-tiny +libbatadv +libjson-c
```

Comment:
*gluon-mesh-batman-adv* is a virtual package. Remove the + sign prefixing the dependency - never use selecting dependencies with virtual packages!

Reply:
done.